iceoryx_posh/test/moduletests/test_roudi_portmanager_client_server.cpp

```diff
@@ -491,6 +491,84 @@ TEST_F(PortManager_test, ServerStateIsForwardedToInterfacePortWhenAlreadyOfferAn
 
 // END forwarding to InterfacePort tests
 
+// BEGIN service registry tests
```

Comment:
Should those tests cases not be added to a different `.cpp`?

Reply:
Well, it's part of the tests to check whether the PortManager adds the ServerPort to the ServiceRegistry. For me this belongs together.

Reply:
... maybe the naming of the comment is poor?

Reply:
What about "BEGIN PortManager interaction tests with service registry"?

Reply:
Sure. The question is just if we want to have another commit and CI run for the change of the comment? I think the fact that the test is int the `PortManager` already shows that it is related to the port manager. It's up to you, if you want I do a commit.

Reply:
Ok, for me to stick to with this comment.